<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Rashmi-Laptop" ignored="0" name="Verify that user is able to sucessfully Logout." tests="1" failures="1" timestamp="2020-09-10T22:19:55 IST" time="0.016" errors="0">
  <testcase name="verifySucessfulLogout" time="0.004" classname="com.clarion.qa.test.LogoutTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.clarion.qa.page.SearchPromisePage.clickOnLogout(SearchPromisePage.java:35)
at com.clarion.qa.test.LogoutTest.verifySucessfulLogout(LogoutTest.java:28)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- verifySucessfulLogout -->
</testsuite> <!-- Verify that user is able to sucessfully Logout. -->
